tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+requires =
+    tox >= 4
+env_list = py38, build-seerep, integration-examples
+skipdist = True
+
+[testenv]
+basepython = python3.8
+description = run tests
+allowlist_externals = bash
+
+[testenv:build-seerep]
+description = build seerep
+commands = bash -c "source ../devel/setup.bash;\
+                    catkin build"
```

Comment:
Something's not right here, the `devel space` contains the final build products, which are not available before building any code. Thus, if you run:

```bash
catkin clean -y 
tox 
```
it results in:

```bash
/usr/bin/bash: ../devel/setup.bash: No such file or directory
```

It generally works in our case due to the Dev-Container running `catkin build` when starting. Switching the order of commands should be a simple solution.